imageio/plugins/ffmpeg.py

```diff
@@ -662,20 +662,39 @@ def run(self):
 
 def parse_device_names(ffmpeg_output):
     """ Parse the output of the ffmpeg -list-devices command"""
+    # Collect device names - get [friendly_name, alt_name] of each
     device_names = []
     in_video_devices = False
     for line in ffmpeg_output.splitlines():
         if line.startswith("[dshow"):
             logger.debug(line)
             line = line.split("]", 1)[1].strip()
             if in_video_devices and line.startswith('"'):
-                device_names.append(line[1:-1])
+                friendly_name = line[1:-1]
+                device_names.append([friendly_name, ""])
+            elif in_video_devices and line.lower().startswith("alternative name"):
+                alt_name = line.split(" name ", 1)[1].strip()[1:-1]
+                if sys.platform.startswith("win"):
+                    alt_name = alt_name.replace("&", "^&")  # Tested to work
+                else:
+                    alt_name = alt_name.replace("&", "\\&")  # Does this work?
+                device_names[-1][-1] = alt_name
             elif "video devices" in line:
                 in_video_devices = True
             elif "devices" in line:
                 # set False for subsequent "devices" sections
                 in_video_devices = False
-    return device_names
+    # Post-process, see #441
+    # prefer friendly names, use alt name if two cams have same friendly name
+    device_names2 = []
+    for friendly_name, alt_name in device_names:
+        if friendly_name not in device_names2:
```

Comment:
An easy way to test the working of the alt names without the need for actually having two equally named cameras:

set the line above to `if False:'